_about/notes/0.8.md

```diff
@@ -8,4 +8,47 @@ type: markdown
 ---
 {% include home.html %}
 
-FILL IN BEFORE RELEASE
+In addition to the usual pile of bug fixes and performance improvements, this release includes the new or
+updated features detailed below.
+
+## Networking
+
+- **Revamped Traffic Management Model**. We're finally ready to take the wraps off our new traffic
+management configuration model. The new model adds many new features and addresses usability issues
+with the prior model.
+[Learn more]({{home}}/docs/tasks/traffic-management-v1alpha3/)
+
+- **Envoy V2**. Pilot now uses the V2 Envoy API to program the proxies in the mesh, which increases
+effective scalability and eliminates spurious 404 errors.
+
+- **Constrain inbound ports**. Restrict the inbound ports in a pod to the ones needed by the apps running inside that pod.
+
+## Security
+
+- **Introducing Citadel**. We've finally given a name to our security component. What was
+formerly known a Istio-Auth or Istio-CA is now called Citadel.
+
+- **Multicluster Support**. [TBD: What should we say?]
+
+- **Authentication Filter**. [TBD: What should we say about the proxy authn filter?]
+
+- **Per-Namespace Authorization Policy**. [TBD]
+
+## Istio Telemetry
+
+- **Self-Reporting**.  Istio components (Mixer, Pilot, Citadel) now produce telemetry that flows through the normal
```

Comment:
for total correctness, we should exclude Citadel from this list -- as it isn't currently fronted by a proxy sending Report()s to Mixer.

Reply:
Yes, please remove Citadel here. For 0.8, Citadel is not using an RPC, it communicates through the secret storage layer.